docs/reporting.md

```diff
@@ -79,31 +81,31 @@ remember that the SRT API call to set the debug log level is:
 POSIX-based systems, and there is a drop-in replacement for it for Windows
 in `common/win/syslog_defs.h`.)
 
-Some applications may use the (not really recommended) C extended API:
+Some applications may use the (not really recommended) an extended C++ API:
```

Comment:
Some applications may use  an extended C++ API (this is not really recommended):